### ISSUE TITLE
Pulmonary mycetoma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6698]
**Preferred term label:**
Pulmonary mycetoma

**Synonyms**
Fungus ball

**Definition (free text, please give PubMed ID)**
A mycetoma is a discrete mass of intertwined hyphae, usually of an Aspergillus species, matted together by mucus, fibrin, and cellular debris colonizing a cavity, usually from prior fibrocavitary disease (eg, tuberculosis or sarcoidosis). A mycetoma may move to a dependent location when the patient changes position and may show an air crescent sign. CT scans may show a spongelike pattern and foci of calcification in the mycetoma.
PMID:18195376

Comment
See Figure 40 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
